test/e2e_kubeadm/bootstrap_token_test.go

```diff
@@ -34,6 +35,21 @@ const (
 	bootstrapTokensCSRAutoApprovalClusterRoleName    = "system:certificates.k8s.io:certificatesigningrequests:nodeclient"
 )
 
+var (
+	bootstrapTokensAllowPostCSR = &authv1.ResourceAttributes{
+		Resource: "certificatesigningrequests",
+		Group:    "certificates.k8s.io",
+		Verb:     "create",
+	}
+
+	bootstrapTokensCSRAutoApproval = &authv1.ResourceAttributes{
```

Comment:
`bootstrapTokensCSRAutoApprovaAttributeCreate`

Reply:
Thanks for your rapid comment. I will amend the PR based on your suggestion and wait for Fabrizio's reply.